_posts/2024-06-20-pyconus-pyopensci-sprints.md

```diff
@@ -0,0 +1,358 @@
+personally interested in. If they are learning data science, they can find data to
+analyze and support their project. For underrepresented groups in STEM (and most
+learners), applied learning resonates because it involves a topic they care about and
+are motivated to pursue. Skills are learned in the process of achieving a meaningful
+outcome.
+
+It's similar to how you can immediately engage a diverse audience in GIS (spatial data) with Google
+Earth by asking them to zoom into the area where they live (place-based learning).
+
+As a volunteer maintainer of Stravalib, I am motivated to work on stravalib
+because I learn in a very applied way.
+
+Sure, a sprint does not have the powerful outcome of a student understanding the
+impacts of climate change on their local tribal lands. But the concept is the same:
+
+> The learning motivation comes from a meaningful outcome that a student wants or cares
+  about.
+
+In leading a sprint, asking that question of "what are your goals for today"
+will help you as a sprint leader to direct their efforts towards a successful path.
+
+### The power of student-directed learning
+
+Every person has different learning preferences. For many years, educators
+taught people the same way: in a classroom, using the same books and approaches.
+However, alternatives to this model allow learners to adapt their environment to their personal and learning goal needs.
+[Student-directed learning](https://scholar.google.com/scholar?q=student-directed+learning&hl=en&as_sdt=0&as_vis=1&oi=scholart)
+is based on the idea that people learn differently and have different needs. By creating
+a hybrid learning environment where students are allowed to pick their learning path and approach, you empowering them to own their experience.
+
+Some learners benefit from hands-on demos, whether in a classroom or at the sprint
+table. They may pick things up quickly or need to watch and digest the demo. Others
+prefer reading quietly on their own, absorbing information, and trying things out until
+they figure it out.
+
+If a sprint is set up correctly, learners can parse through available issues. Carefully
+tagged issues direct them to ones that are beginner-friendly. These issues should not
+require extensive Git and GitHub expertise, ensuring a good first experience and an
+early win to build confidence.
+
+Contributors at the sprint can decide if they need direction, mentorship, or just a
+short tutorial/guidebook for making their first contribution using the GitHub
+interface.
+
+### Collaborative git and GitHub lessons
+
+This past year, I was awarded the BSSw fellowship in which I proposed lessons on these lesser known collaborative,
+git and GitHub skills - the same skills that will support people attending sprints! I am looking forward to having these lessons on hand,for next year's sprints, to support those who are still fighting to learn git and GitHub (it can be an uphill process!).
```

Comment:
```suggestion
Git and GitHub skills—the same skills that will support people attending sprints! I am looking forward to having these lessons on han ,for next year's sprints, in order to support those who are still fighting to learn Git and GitHub (it can be an uphill process!).
```